client/src/components/Ticketing/ticketingmanager/udash_navbar.tsx

```diff
@@ -86,7 +86,7 @@ const Udash_nav = () => {
           <div className={showMobileMenu ? 'w-full h-full absolute z-40  transform  translate-x-0 ' : 'w-full h-full absolute z-40 transform -translate-x-full'} id="mobile-nav">
             <div className="bg-gray-800 opacity-50 absolute h-full w-full md:hidden" onClick={() => setShowMobileMenu(!showMobileMenu)} />
             <div className="absolute z-40 sm:relative w-[14rem] md:w-96 shadow  bg-zinc-900 md:hidden transition duration-150 ease-in-out h-full">
-              <div className="flex flex-col justify-between h-full w-full">
+              <div className="flex flex-col justify-between h-full w-full overflow-auto">
```

Comment:
I think there would also need to be an `overflow-auto` on line 36 so that overflow is accounted for on the non-mobile view.

Reply:
Yeah I'm not sure why I added this, let me double check

Reply:
It's good you added it here, but I think you also need it in the non-mobile version as well